cmd/root.go

```diff
@@ -36,6 +43,14 @@ func Execute(version string, commit string, date string) {
 }
 
 func init() {
+	rootCmd.PersistentFlags().StringVarP(&ManifestPath, "manifest-path", "m", ManifestPath, "Specify the path and name for the manifest file")
+	
+	viper.BindPFlag("manifest-path", rootCmd.PersistentFlags().Lookup("manifest-path"))
+	
+	viper.SetConfigName(".openfeature")
+	viper.AddConfigPath(".")
+	viper.ReadInConfig()
+
 	rootCmd.AddCommand(generate.Root)
 	rootCmd.AddCommand(versionCmd)
```

Comment:
is our plan to register the new manifest cmd here at root?

Reply:
Yeah, but we'll need to figure out if they should be top level commands or nested.

Should we use:

- `openfeature manifest init`
- `openfeature manifest compare`
- `openfeature manifest sync`

or

- `openfeature init`
- `openfeature compare`
- `openfeature sync`

I'd like to have an agreement on this structure because it will be difficult to change later.

Reply:
Yeah, agreed, once they are published commands, changing them is difficult on users.

These features feel very core to the tool, so I lean towards the simpler `openfeature init` example. 

I do appreciate that the `openfeature manifest init` approach is less risky, but the assurance is at the cost of verbosity. 

I suppose the right answer depends on the overall mission of this CLI tool. If it's pretty focused on initializing, comparing, and syncing these flags; then I think the less verbose answer could be the right fit. If it's mission could develop into something more broad, I'd start leaning more towards verbosity to support the long term.

Reply:
I decided to keep manifest interaction at the top level. For now, I've only added `init`.